sdk/python/tests/integration/offline_store/test_universal_historical_retrieval.py

```diff
@@ -330,3 +352,62 @@ def test_historical_features(environment, universal_data_sources, full_feature_n
         .reset_index(drop=True)
     )
     assert_frame_equal(actual_df_from_df_entities_for_table, table_from_df_entities)
+
+
+def response_feature_name(feature: str, full_feature_names: bool) -> str:
+    if (
+        feature in {"current_balance", "avg_passenger_count", "lifetime_trip_count"}
```

Comment:
seems like you only ever call this with conv_rate?

Reply:
could remove other if blocks if they're not needed

Reply:
Yeah I copied over the one from test_universal_online. Removing.